src/test/java/Pages/HomePage.java

```diff
@@ -0,0 +1,41 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+
+By firstPlaylist = By.cssSelector("#playlists > ul > li:nth-child(4) > a");
+
+    public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+
+    }
+    public void doubleClickFirstPlaylist(){
+       actions.doubleClick(driver.findElement(By.cssSelector("#playlists > ul > li:nth-child(4) > a")));
```

Comment:
We can make the xpath or css the class variable (Locator)

![image](https://user-images.githubusercontent.com/104761482/226164046-0c0cc289-de11-4091-9c11-54626a6b4aae.png)